paint/preprocessing/binary_extractor.py

```diff
@@ -179,7 +179,7 @@ def convert_to_h5(
         # To maintain consistency, we cast the west direction to east direction.
         canting_e[:, 0] = -canting_e[:, 0]
 
-        # extract deflectometry data and save
+        # extract deflectometry preprocessing and save
```

Comment:
Change "preprocessing" back to "data" (cannot make commit suggestion)? Also add full stop at the end of the comment sentence.